packages/salesforcedx-vscode-apex/package.json

```diff
@@ -356,7 +356,7 @@
       {
         "id": "apex",
         "aliases": [
-          "Apex",
+          "$(sync~spin) Apex",
```

Comment:
was playing around here trying to see if I could programmatically change the alias afterwards, I couldn't find it exposed anywhere to update

Reply:
@lukeis If this cannot be updated, should this change be reverted?

Reply:
correct, was just trying to show Work In Progress. This PR isn't really ready for review :)